hazelcast/src/main/java/com/hazelcast/map/impl/operation/PutAllOperation.java

```diff
@@ -128,7 +128,7 @@ private Object putToRecordStore(Data dataKey, Data dataValue) {
         if (hasMapListener) {
             return recordStore.put(dataKey, dataValue, DEFAULT_TTL);
         }
-        recordStore.set(dataKey, dataValue, DEFAULT_TTL);
```

Comment:
I wish Java had named arguments instead of the module system. Reading a boolean value really provides no insight in its meaning.

Reply:
Me too wishing this. I have tried to introduce a enum to make this more readable, but it was too awkward, so I removed it.